client/me/purchases/manage-purchase/payment-info-block.tsx

```diff
@@ -69,6 +69,10 @@ export default function PaymentInfoBlock( { purchase }: { purchase: Purchase } )
 		);
 	}
 
+	if ( purchase.isInAppPurchase ) {
+		return <PaymentInfoBlockWrapper>{ translate( 'In-App Purchase' ) }</PaymentInfoBlockWrapper>;
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( 'Apple In App Purchase' )` *ES Score: 8*

Reply:
This could be an Apple or Android in-app purchase 😞 So whilst we could make this app store-specific, it would still add another translation and it might be useful to have the generic version if one is being added anyway.